iOS/MMIWG2S/Utilities/Network.swift

```diff
@@ -46,18 +44,29 @@ class Network {
                 }
                 request.httpBody = parameterString.data(using: .utf8)
             case .multipartForm:
+                request.setValue("multipart/form-data; boundary=\(boundary)", forHTTPHeaderField: "Content-Type")
                 var multiPartData = Data()
-                guard let data = data as? [String:(String,Data)] else {
+                guard let data = data as? [String:(String, Data)] else {
                     log.error("Incorrect format for multipart form")
                     completionHandler("incorrect data format", nil)
                     return
                 }
-                metadata.forEach {
```

Comment:
Nice! Since we no longer need this could u remove the parameter variable? thanks!

Reply:
Removed the metadata param